src/Components/ErrorBoundary/ErrorBoundary.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Text, Center } from '@chakra-ui/react';
+
+class ErrorBoundary extends React.Component {
```

Comment:
rewrite it to a function component please
class components aren't being used anywhere anymore